docs/Directory.Packages.props

```diff
@@ -1,6 +1,3 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Packages.props, $(MSBuildThisFileDirectory)..))" />
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs/getting-started-console The readme here can be now simplified as there is no need to install Ms.Ext.log latest, as it'll be automatically done now...

Reply:
I know. I intend to keep it as is because most folks who read the document are using existing stable versions. I plan to make a change when the next stable release is shipped.

Reply:
And there are other changes we would want to make, e.g. #4902.

Reply:
IMO you could use conditional compilation for .NET8/force update packages only to examples folder.